src/dotnet/APIView/APIViewWeb/Controllers/CopilotCommentsController.cs

```diff
@@ -0,0 +1,91 @@
+using System.Text.Json;
+using System.Threading.Tasks;
+using APIViewWeb.Managers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace APIViewWeb.Controllers
+{
+    public class CopilotCommentsController : Controller
```

Comment:
We should add an authorization filter to ensure only calls with a access token in header is authorized to run these APIs.